content/advanced/modules/create-example/index.md

```diff
@@ -0,0 +1,304 @@
+---
+title: "Create an Example Module"
+weight: 320
+---
+
+### Overview
+
+In this module, you will follow steps to register a sample CloudFormation Module as a private extension with the AWS CloudFormation registry in your AWS account.
+
+For this example we will be creating a Module that deploys an entire VPC, including associated resources with defaults set. This example has been chosen to show how something as complex as a VPC can be defined by a central team in a best practice way and then consumed easily by other teams.
+
+### Topics Covered
+
+By the end of this lab, you will be able to:
+
+* understand key concepts to leverage when you develop a Module;
+* use the [CloudFormation Command Line Interface (CLI)](https://docs.aws.amazon.com/cloudformation-cli/latest/userguide/what-is-cloudformation-cli.html) to create a new project and submit the resource type as a private extension to the CloudFormation registry in your AWS account;
+* understand how to consume the Module in CloudFormation Templates.
+
+### Start Lab
+
+#### Sample Module Walkthrough
+
+Let's get started! Create a new directory and then issue the following command from inside that directory:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+mkdir module
+cd module
+cfn init
+:::
+
+You will be prompted to answer some questions. Supply the answers as shown below:
+
+```
+Initializing new project
+Do you want to develop a new resource(r) or a module(m) or a hook(h)?.
+>> m
+What's the name of your module type?
+(<Organization>::<Service>::<Name>::MODULE)
+>> CFNWORKSHOP::EC2::VPC::MODULE
+Directory  /home/user/cfn101-workshop/module/fragments  Created 
+Initialized a new project in /home/user/cfn101-workshop/module
+```
+
+Let's take a look at what the command created in the directory structure:
+
+* `fragments/`: contains an auto generated `sample.json` CloudFormation template file;
+* `.rpdk-config`: this is the config file that stores the details you supplied when you ran the init command above;
+* `rpdk.log`: a log file for the actions carried out by the cfn cli;
+
+
+Let's first clean up a little. You will be using YAML format for this workshop so go ahead and delete the `sample.json` file, you will not be needing it:
+
+:::code{language=shell showLineNumbers=false showCopyAction=true}
+rm fragments/sample.json
+:::
+
+CloudFormation Modules are created using a standard CloudFormation Template, just like those you have been creating already in this workshop. However, Modules can use a single template file only no nested stacks are supported.
```

Comment:
Replace the last sentence here with "A module is created with a single template file. Nested stacks are not supported".